test/test_ops.py

```diff
@@ -271,6 +271,10 @@ def test_variant_consistency_eager(self, device, dtype, op):
                 if variant in inplace_ops and skip_inplace:
                     continue
 
+                if not callable(variant):
```

Comment:
IIRC as per the discussion in previous PR, plan was to update following lines if the operator is not callable.
https://github.com/pytorch/pytorch/blob/1b4bb3691cbddfea45805b8459292842e5b23f17/torch/testing/_internal/common_methods_invocations.py#L191-L195

This way we won't have to update anything in `test_ops` as they would get filtered at following lines automatically.
https://github.com/pytorch/pytorch/blob/1b4bb3691cbddfea45805b8459292842e5b23f17/test/test_ops.py#L238

https://github.com/pytorch/pytorch/blob/1b4bb3691cbddfea45805b8459292842e5b23f17/test/test_ops.py#L308-L309


I think you might want to discuss and finalise the approach once.

@anjali411 @mruberry

Reply:
yeah this sounds good!

Reply:
yup as @kshitij12345 mentioned, we should set the method_variant to `None` if the operator is not callable!